Pagina registro

JLeonN opened the pull request.

JLeonN committed:
Cosas pendiente

Cambiando mayúscula por minúscula.
Borrando comentarios.

JLeonN committed:
Confirmar contraseña

JLeonN committed:
Registro pronto

El registro está casi listo
Solo falta la verificación de contraseña dos veces.

Además, añadí un comentario a un console.log(); que me confundió por un rato.
Ahora todo está más claro en console.log()!  💻🔍

JLeonN committed:
Manejo de error

francosang reviewed (dismissed):
Vamoooo @JLeonN !

Ahora acordate de seguir con el componente global:
 - https://trello.com/c/81rfxJuy/75-componente-global-de-sesion

francosang commented on frontend/src/Pagina/InicioDeSesión/Registro.jsx:
```diff
@@ -1,6 +1,33 @@
+  const [errorVacio, setErrorVacio] = useState(false);
+  const navegar = useNavigate();
+
+  const btnRegistro = async () => {
+    if (email == "" || contraseña == "") {
+      setErrorVacio(true);
+    } else {
+      const reg = {
+        email: email,
+        password: contraseña,
+      };
+      try {
+        await registro(reg);
+        navegar("/");
+        
+      } catch (error) {
+        // console.log(error.response.status);
+        setErrorRegistro(true);
+      }
+    }
+  };
```
Al principio de esta funcion hay que poner los errores en false, porque si no al hacer click se quedan los errores anteriores.

JLeonN replied:
Esto después explícamelo cuando estemos conectados

francosang reviewed (commented):
Te dejé un comentario, fijate si lo queres hacer.

francosang commented on frontend/src/Pagina/InicioDeSesión/Registro.jsx:
```diff
@@ -25,17 +52,25 @@ const Registro = () => {
+            onChange={(e) => setContraseña(e.target.value)}
           />
         </div>
+        {/* <div className="campoLogin">
+          <label htmlFor="password">Contraseña:</label>
+          <input
+            id="password"
+            type="password"
+            name="contrasena"
+            placeholder="Repita su contraseña"
+            required
+            value={contraseña}
+            onChange={(e) => setContraseña(e.target.value)}
+          />
+        </div> */}
```
Esto no lo vas a dejar?

JLeonN replied:
Esto lo voy hacer ahora 
y lo dejo pronto en esta PR

JLeonN committed:
Validar la contraseña

Me falta dar un mensaje que avise al usuario cuando las contraseñas no coinciden. 🛡️

JLeonN committed:
Mensaje - Contraseñas no coinciden

JLeonN committed:
Comentarios

JLeonN committed:
Errores en false

francosang reviewed (approved):
Vamoooo!
